app/editor/src/features/content/form/ContentSummaryForm.tsx

```diff
@@ -165,15 +165,17 @@ export const ContentSummaryForm: React.FC<IContentSummaryFormProps> = ({
                 label="Event of Day Category"
                 width={FieldSize.Medium}
                 options={categoryOptions}
-                clearValue={[]}
                 value={
                   values.categories.length
                     ? categoryOptions.find((c) => c.value === values.categories[0].id) ?? []
                     : []
                 }
                 onChange={(e: any) => {
                   // only supports one at a time right now
-                  const value = categories.find((c) => c.id === e.value);
+                  let value;
+                  if (!!e?.value) {
+                    value = categories.find((c) => c.id === e.value) ?? [];
```

Comment:
This is messy I can't understand what's going on.
The following should return an array with one item, or an empty array.
```
const value = categories.filter((c) => c.id === e.value);
setFieldValue('categories', value);
```

Reply:
The above throws an error when clearing the field, will try to clean it up though!